ironfish/src/network/peers/peer.ts

```diff
@@ -211,6 +213,7 @@ export class Peer {
 
   constructor(
     identity: Identity | null,
+    metrics: MetricsMonitor,
```

Comment:
Can this go into the options object below?

Reply:
yep, added. I also am removing this `= {}` at the end of the options parameter. It looks like what that's saying is that if no argument is passed in for options then we default to an empty object? I don't see that being used anywhere and it also just seems like a strange thing to do. Hopefully I am not missing anything there though